backend/molgenis-emx2/src/test/java/org/molgenis/emx2/TestTypeUtils.java

```diff
@@ -63,4 +63,11 @@ public void testCommaInCsvString() {
     assertEquals("value with, comma", result[0]);
     assertEquals("and, another", result[1]);
   }
+
+  @Test
+  public void testDate() {
```

Comment:
please add the reason for this test , what part /feature does it test , 'testDate' is to general is this test does not test the java Date object but rather s specific part of the string the string format